abstract_specification.md

```diff
@@ -284,6 +284,7 @@ Parameters:
 | get soma_type                           | Returns the constant "SOMADataFrame"                                           |
 | get schema -> Arrow.Schema              | Return data schema, in the form of an Arrow Schema                             |
 | get index_column_names -> [string, ...] | Return index (dimension) column names.                                         |
+| get count -> int                        | Return the number of rows in the SOMADataFrame.                                |
```

Comment:
This probably makes sense to be implemented as `__len__` in Python (in addition to the explicit `count` accessor).

Reply:
absolutely!